### DISCUSSION TITLE
Make Explore shortened link expiration timeout configurable

### DISCUSSION THREAD

**Discussion by Mario-Hofstaetter** [grafana/grafana#37331] [Explore]
## What would you like to be added

It would be nice to be able **to configure a larger expiration time than 7 days for shortended links created in Explore.**

## Why is this needed

Explore Links may become **very _very_ long** (especially LogQL queries), increased by the URL escaping of many characters in Queries. Using such links in IM / E-Mail / Ticket systems is not pleasant.

Which is why you added short-links. But these appear to expire after 7 days of inactivity (see below).
When adding a link to a bug report ticket system, it may not be needed for days or even weeks and therefore expire too soon.

## Current state

In <https://grafana.com/docs/grafana/latest/explore/#share-shortened-link> it says

> A shortened link that is never used will automatically get deleted after seven (7) days.

Since it is nowhere found in <https://grafana.com/docs/grafana/latest/administration/configuration/> it does not appear to be configurable? Or is it just not documented yet? :-)

## PS

Not sure if "General" is the correct category for this feature request.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #83009. Feel free to continue the discussion around this there. Thank you!
